### PR TITLE
Migrate relevant 3.2 commits to the new branch structure

### PR DESCRIPTION
**Pull request by handrews**
This includes the few commits that were:

* part of `v3.2.0-dev`
* **not** part of `v3.1.1-dev`
* **not** about removing the special Path Item Object `$ref`, or undo-ing that removal (let's just skip the whole thing...)

These were handled with `git format-patch` and `git am`, like we have been doing to sync different release lines.  In some cases, I noticed that the anchors for the new fields were incorrect and fixed them as if they were `git am` merge conflicts (so there are not separate commits).

So this should be as close to what's on `v3.2.0-dev` in the `versions/3.2.0.md` file as we need it to be.  But not identical.

### CODE REVIEW

File: src/oas.md [hunk 1]
```diff
@@ -3932,10 +3936,14 @@ Configuration details for a supported OAuth Flow
...
 | <a name="oauth-flow-refresh-url"></a>refreshUrl | `string` | `oauth2` | The URL to be used for obtaining refresh tokens. This MUST be in the form of a URL. The OAuth2 standard requires the use of TLS. |
 | <a name="oauth-flow-scopes"></a>scopes | Map[`string`, `string`] | `oauth2` | **REQUIRED**. The available scopes for the OAuth2 security scheme. A map between the scope name and a short description for it. The map MAY be empty. |
+| <a name="oauth-flow-ciba-delivery-modes"></a>cibaDeliveryModes | Array[`string`] | `oauth2` (`"ciba"`) | **REQUIRED**. JSON array containing one or more of the following values: `poll`, `ping`, and `push`. |
+| <a name="oauth-flow-ciba-signing-algs"></a>cibaSigningAlgs | Array[`string`] | `oauth2` (`"ciba"`) |  JSON array containing a list of the JWS signing algorithms (alg values) supported by the OP (OpenIdConnect Provider) for signed authentication requests |
+| <a name="oauth-flow-ciba-user-code"></a>cibaUserCode | `boolean` | `oauth2` (`"ciba"`) |  Boolean value specifying whether the OP supports the use of the user_code parameter, with true indicating support. If omitted, the default value is false. |
```
Comment (ralfhandl) [inline src/oas.md:3946]: I know this is just recreating state already present in branch `v3.2.0-dev`, but what does "OP" stand for? Probably not "On-Premises" 😎

Comment (ralfhandl) [inline src/oas.md:3946]:
```suggestion
| <a name="oauth-flow-ciba-signing-algs"></a>cibaSigningAlgs | Array[`string`] | `oauth2` (`"ciba"`) |  JSON array containing a list of the JWS signing algorithms (alg values) supported by the OpenIdConnect Provider for signed authentication requests |
| <a name="oauth-flow-ciba-user-code"></a>cibaUserCode | `boolean` | `oauth2` (`"ciba"`) |  Boolean value specifying whether the OpenIdConnect Provider supports the use of the user_code parameter, with true indicating support. If omitted, the default value is false. |
```
Avoid abbreviation that is only used once

**Review by ralfhandl — COMMENTED** [review]
+1, one nit that I could address in a separate PR if I absolutely have to 😎

**Review by mikekistler — DISMISSED** [review]
Looks good. 👍

**Comment by handrews** [comment]
@ralfhandl I merged your nit even though this was a port :-)  Could you and @mikekistler please re-approve?

**Review by ralfhandl — DISMISSED** [review]

**Comment by ralfhandl** [comment]
@handrews Gladly, thanks!

**Comment by lornajane** [comment]
Changed to draft status to allow more discussion of the best way to add these and possibly other changes. There's more discussion on #4106

**Comment by karenetheridge** [comment]
Looks like there would be schema changes to accompany these?

**Comment by handrews** [comment]
In the TDC Call we decided to defer CIBA to 3.3 based on #4106 (if that somehow comes together quickly enough, it can get on the 3.2 train, but we won't hold up 3.2 since we're going to do 3.3 immediately after per #4210).  I'm going to remove the CIBA commit(s) and take this out of draft.

@karenetheridge since this is just porting pre-existing work, please feel free to make a follow-up PR for the relevant schema changes on the `v3.2-dev` branch.  I don't want to introduce new work in this PR.

File: scripts/md2html/build.sh [hunk 2]
```diff
@@ -41,7 +41,7 @@ cp -p ../../node_modules/respec/builds/respec-w3c.* ../../deploy/js/
 latest=`git describe --abbrev=0 --tags`
 latestCopied=none
 lastMinor="-"
-for filename in $(ls -1 ../../versions/[23456789].*.md | sort -r) ; do
+for filename in ../../versions/3.2.0.md ; do
```
[the remark this answers is not in the document]
Comment (handrews) [reply scripts/md2html/build.sh:44]: I don't remember this change being part of this PR at all, let me figure out what happened and update.

[the remark this answers is not in the document]
Comment (handrews) [reply scripts/md2html/build.sh]: @ralfhandl should be fixed now!

**Review by ralfhandl — APPROVED** [review]

**Review by lornajane — APPROVED** [review]

**Pull request merged**